### ISSUE TITLE
Share functionality based on existing export

### ISSUE THREAD

**Issue by holzenburg** [superfeedr/subtome#68]
In a recent email conversation with Julien, we discussed possibilities to implement a share functionality. Background: I launched the feedshare.net service some days ago. Julien contacted me and I implemented the SubToMe button. The next idea that came up was to have a direct import flow on feedshare.net – or even an „Share with ...“ function on SubToMe.

There is the export via http://subtome.com/#/export. The subscription data is stored in localStorage, but only accessible by subtome.com.

To summarize Juliens attempt:
- Create another page like /#/export, maybe /#/share
- Other application (like feedshare.net) calls that with a callback url.
- User gets redirected back with the subscription data as a JSON list.

In general, I think that's a great way to do it. I would add something, though:
- That share-page should return to the callback via POST to avoid URL length issues.
- It could/should accept a parameter to specify the return method (JSON or OPML)
